specs/waku/v2/waku-message.md

```diff
@@ -76,6 +77,10 @@ topic, data, nonce`.
 Since Waku v2 is using libp2p PubSub, some of these fields can be dropped. The previous `topic`
 field corresponds to `contentTopic`. The previous `data` field corresponds to the `payload` field.
 
+# Security Consideration
+
+In Waku, the confidentiality, integrity, and authenticity of the data must be addressed at the `WakuMessage` level.  That is, the `payload` shall be encrypted or signed properly to meet the application-specific privacy needs.  
```

Comment:
This phrasing seems a bit weird to me considering we are in WakuMessage spec already. There is also a difference between version 0 and version 1, so would be useful to clarify that. Let's get it merged though...

Reply:
Sure, merged. I took note of it and will put it on our today's call agenda to discuss further.